pages/projects.js

```diff
@@ -0,0 +1,357 @@
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import Checkbox from '@material-ui/core/Checkbox';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import DeleteIcon from '@material-ui/icons/Delete';
+import FilterListIcon from '@material-ui/icons/FilterList';
+import Switch from '@material-ui/core/Switch';
+import Link from '@material-ui/core/Link';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  toolbar: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  leftToolbar: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+  },
+});
+
+function createData(id, name, shared, teamMembers, updatedAt, createdAt) {
+  return {
+    id,
+    name,
+    shared,
+    teamMembers,
+    updatedAt,
+    createdAt,
+  };
+}
+
+const rows = [
+  createData(
+    1,
+    'Sign in component',
+    true,
+    'Abid, Emily, Blake and Dan',
+    'Thursday, August 14th 2021',
+    'Thursday, August 14th 2021'
+  ),
+  createData(
+    2,
+    'Profile page',
+    true,
+    'Abid, Emily, Blake and Dan',
+    'Thursday, August 14th 2021',
+    'Thursday, August 14th 2021'
+  ),
+  createData(
+    3,
+    'Logout screen',
+    true,
+    'Abid, Emily, Blake and Dan',
+    'Thursday, August 14th 2021',
+    'Thursday, August 14th 2021'
+  ),
+  createData(
+    4,
+    'Image screen',
+    false,
+    'Abid, Emily, Blake and Dan',
+    'Thursday, August 14th 2021',
+    'Thursday, August 14th 2021'
+  ),
+];
+
+function descendingComparator(a, b, orderBy) {
+  if (b[orderBy] < a[orderBy]) return -1;
+  if (b[orderBy] > a[orderBy]) return 1;
+  return 0;
+}
+
+function getComparator(order, orderBy) {
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+const headCells = [
+  {
+    id: 'name',
+    numeric: false,
+    disablePadding: true,
+    label: 'Project',
+  },
+  {
+    id: 'shared',
+    numeric: false,
+    disablePadding: false,
+    label: 'Shared',
+  },
+  {
+    id: 'teamMembers',
+    numeric: false,
+    disablePadding: false,
+    label: 'Team Members',
+  },
+  {
+    id: 'updatedAt',
+    numeric: false,
+    disablePadding: false,
+    label: 'Updated At',
+  },
+  {
+    id: 'createdAt',
+    numeric: false,
+    disablePadding: false,
+    label: 'Created At',
+  },
+];
+
+function EnhancedTableHead(props) {
```

Comment:
I think we should make this into a separate component to modularize the code and promote maintainability.

Reply:
done